includes/widgets/star-rating.php

```diff
@@ -89,6 +101,17 @@ protected function register_controls() {
 			]
 		);
 
+		if ( Plugin::$instance->widgets_manager->get_widget_types( 'rating' ) ) {
+			$this->add_deprecation_message(
+				'3.17.0',
+				esc_html__(
+					'You are currently editing a Star Rating widget in its old version. Drag a new Rating widget onto your page to use a newer version, providing better capabilities.',
+					'elementor'
+				),
+				'rating'
+			);
+		}
```

Comment:
Shouldn't this be a control of type `RAW_HTML`?

Reply:
We have used this method for the other widgets, see e.g. the accordion widget: https://github.com/elementor/elementor/blob/11eab6625436289eedb5b4c6e158363bdddfb81e/includes/widgets/accordion.php#L129.